Lesson33/props-drilling-app/Gokhan/src/components/Navbar/index.jsx

```diff
@@ -0,0 +1,23 @@
+import './styles.css';
+import { loggedInUser, loggedOutUser } from '../../data';
+
+export const Navbar = ({ isLoggedInUser, onAccountClick, onHomeClick, onSignOut, onSignIn }) => {
+	const user = isLoggedInUser ? loggedInUser : loggedOutUser;
```

Comment:
This statement makes no sense. You already have a boolean `isLoggedInUser` passed to the component as props that defines if user is logged in or not, just use it.